lib/builder-browserify.js

```diff
@@ -69,6 +69,12 @@ module.exports = function (files, config) {
       cb(null, src, map)
     })
   }
+
+  bundle.close = function () {
+    watchifyBundler.close()
+  }
+
+  return bundle
```

Comment:
`control-app.js` uses this bundler and needs to call `.close()` to close its underlying `watchify` instance. Also needed to make `tape` not hang.